pkgs/applications/science/electronics/pcb/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl, pkgconfig, gtk, bison, intltool, flex, netpbm, imagemagick, dbus, xlibsWrapper, mesa, desktop_file_utils,  shared_mime_info, tcl, tk, gnome, pangox_compat, gd, xorg }:
+
+stdenv.mkDerivation rec {
+  name = "pcb-${version}";
+  version = "20140316";
+
+  src = fetchurl {
+    url = "http://ftp.geda-project.org/pcb/pcb-20140316/${name}.tar.gz";
+    sha256 = "0l6944hq79qsyp60i5ai02xwyp8l47q7xdm3js0jfkpf72ag7i42";
+  };
+
+  buildInputs = [ pkgconfig gtk bison intltool flex netpbm imagemagick dbus xlibsWrapper mesa desktop_file_utils tcl shared_mime_info tk gnome.gtkglext pangox_compat gd xorg.libXmu ];
+
+  meta = with stdenv.lib; {
+    description = "PCB: Printed Circuit Board editor";
+    homepage = http://pcb.geda-project.org/;
+    maintainers = with maintainers; [ mog ];
+    platforms = platforms.linux;
+    license = licenses.gpl2;
+    priority = 10;
```

Comment:
Wide character in die at /nix/store/jjgyvh88yw37gg5ad5lj1z0qdv31d7jy-buildenv.pl line 79.
collision between ‘/nix/store/yndn1yfxdh2px9876mv22zhn61mwank0-pcb-20140316/share/applications/mimeinfo.cache’ and ‘/nix/store/q8d3c4fhjr1sxhbljdcxfm7s19zjlvzj-hexchat-2.10.2/share/applications/mimeinfo.cache’; use ‘nix-env --set-flag priority NUMBER PKGNAME’ to change the priority of one of the conflicting packages
builder for ‘/nix/store/sc53w9yvfndcq0nhxwa085acjnczx6n3-user-environment.drv’ failed with exit code 255
error: build of ‘/nix/store/sc53w9yvfndcq0nhxwa085acjnczx6n3-user-environment.drv’